.github/workflows/ci.yml

```diff
@@ -186,15 +199,14 @@ jobs:
         run: |
           composer update --no-ansi --no-interaction --no-progress --with-dependencies
           composer show
-      - name: "Start MySQL"
-        run: "sudo /etc/init.d/mysql start"
+      - name: "Set up TYPO3"
+        run: >
+          .Build/vendor/bin/typo3cms install:setup --no-interaction --site-setup-type="site"
+          --database-user-name="${DATABASE_USER}" --database-host-name="${DATABASE_HOST}"
+          --database-port="${{ job.services.mysql.ports[3306] }}" --database-name="${DATABASE_NAME}"
+          --admin-user-name="admin" --admin-password="password" --site-name="Test installation";
```

Comment:
In case you copied this from my PHPUnit extension: I only do it there to get the console commands to work (as the PHPUnit extension provides a `phpunit:run` command). This should not be required for getting the usual functional tests to run.